docs/pages/blog/material-ui-2024-updates.md

```diff
@@ -0,0 +1,46 @@
+---
+title: Material UI 2024 end-of-year updates
+description: A look through Material UI during 2024, and a sneak peek of what's to come.
+date: 2024-12-11T00:00:00.000Z
+authors: ['diegoandai']
+tags: ['Material UI', 'Product']
+manualCard: true
+---
+
+As the end of the year approaches, we want to take some time to review some of this year's highlights for Material UI and update you on what the community can expect moving forward.
+
+## Material UI v6 release and support
+
+We released Material UI v6 in August, offering new features, fixes, and a smoother migration experience. So far, almost a million users have successfully migrated.
+
+The reception for the new version has been very positive, and we hope many more users can migrate in the upcoming months.
+
+Our focus has been on responding to v6-related issues to support your migration. We thank everyone in the community who has helped by reporting or fixing problems and encourage everyone else to migrate. We are happy and ready to help!
+
+## React 19 support
+
+As of today, Material UI v5 and v6 support React 19. This has been one of our main focuses during the second half of 2024.
+
+Next up, we'll update Material UI's dependencies and type versions once React 19 is stable.
+
+We hope you start trying it, and we're very grateful to the community members who have been and continue to test and report issues.
+
+## Material Design 3
+
+A year ago, we shared that we expected to have a Material Design 3 implementation by the end of 2024. Due to the efforts mentioned above, this has not been the case yet.
+
+Updating the look and feel in such a big way is very complex. We want to provide the best experience for everyone and we're currently discussing and working on the best way to achieve it. This will be one of our main focuses for 2025.
+
+## Looking towards 2025
+
+We're planning a new major release for the first half of 2025: Material UI v7.
+
+This version will fix issues that require breaking changes, including the long-awaited ESM support improvements, and other small fixes.
+
+Note that, while this major release won't contain visual updates since its only focus is fixing bugs, we'll keep exploring the best way to update the look and feel in parallel. We'll continue sharing any details as soon as we reach any decisions.
```

Comment:
here's how I'd construct this section. Could drop the last clause there after the "—" if we don't want to say anything about v8 at all.

```suggestion
This version will fix issues that require breaking changes, including the long-awaited ESM support improvements, and other small fixes.
While v7 won't contain visual updates since its sole focus is fixing bugs, rest assured that we will be working on updating the look and feel in parallel—that work could potentially become Material UI v8, but right now it's too early to say for sure.
```